### PR TITLE
V2 with python

### PR DESCRIPTION
**Pull request by matks**
Thanks @PierreRambaud for the help

### CODE REVIEW

File: core-weekly-generator.py [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+    #
+    # :param str url: URL string
+    #
+    def extract_repository(self, url):
+        m = re.search('github.com/PrestaShop/(.+?)/', url)
+        if m:
+            if m.group(1) in PROJECTS.keys():
+                # map repository with a project name
+                return PROJECTS[m.group(1)]
+            else:
```
Comment (PierreRambaud) [inline core-weekly-generator.py]:
```suggestion
```

File: core-weekly-generator.py [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+                return m.group(1)
```
Comment (PierreRambaud) [inline core-weekly-generator.py]:
```suggestion
            return m.group(1)
```

File: coreteam.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    'david-piatek',       # David P.
+    'yelnatss',           # Marvin S.
+    'CaptainYouz',        # Anas M.
+    'Matt75',             # Matthias R.
+    'SimonGrn',           # Simon G.
+    'mbadrani',           # Mehdi B.
+    'boubkerbribri',      # Boubker B.
+    'matthieu-rolland',   # Matthieu R.
+    'Progi1984',          # Franck L.
+
```
Comment (Progi1984) [inline coreteam.py]: @matks It lacks Robin (QA Team) ?

File: coreteam.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    'Joukz',              # Julien G.
+    'david-piatek',       # David P.
+    'yelnatss',           # Marvin S.
+    'CaptainYouz',        # Anas M.
+    'Matt75',             # Matthias R.
+    'SimonGrn',           # Simon G.
+    'mbadrani',           # Mehdi B.
+    'boubkerbribri',      # Boubker B.
+    'matthieu-rolland',   # Matthieu R.
+    'Progi1984',          # Franck L.
```
Comment (eternoendless) [inline coreteam.py:39]: Shouldn't we delete the ones who no longer work in it?

File: coreteam.py [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    'prestashop-specs' : 'PrestaShop Specifications',
+    'docs' : 'Changes in developer documentation',
+    'TranslationToolsBundle' : 'TranslationTools Bundle',
+    'QANightlyResults' : 'QA nightly results',
+    'php-coding-standards' : 'PHP Coding Standards',
+    'ps_facetedsearch' : 'Faceted search module',
+    'gamification' : 'Gamification module',
+    'autoupgrade' : 'Auto Upgrade module',
+    'classic-rocket' : 'Classic-rocket theme',
+    'prestashop-ui-kit' : 'Prestashop UI Kit',
```
Comment (eternoendless) [inline coreteam.py:57]: Aren't we missing like 80 repos?

File: coreteam.py (hunk 4 above)
Comment (matks) [reply coreteam.py:39]: Maybe. But if we do this, the generator cannot be used on older PRs.

File: coreteam.py (hunk 5 above)
Comment (matks) [reply coreteam.py:57]: The goal of this list is to speed up the polishing work done after the generation as these are the repositories "most often updated". I dont think we should put all the repositories in it, only the ones most often updated. This list can grow over time.

File: coreteam.py (hunk 3 above)
Comment (Progi1984) [reply coreteam.py]: :ok_hand:

**Review by Progi1984 — APPROVED** [review]

**Pull request merged**